src/main/kotlin/blackjack/domain/card/CardFactory.kt

```diff
@@ -0,0 +1,7 @@
+package blackjack.domain.card
+
+import blackjack.domain.Card
+
+interface CardFactory {
+    fun createCards(): MutableList<Card>
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    fun createCards(): List<Card>
```